quantmsio/core/common.py

```
@@ -0,0 +1,137 @@
+    "PIF": "parent_ion_fraction",
+}
+
+MAXQUANT_FEATURE_MAP = {
+    "Sequence": "sequence",
+    "Proteins": "mp_accessions",
+    "Leading proteins": "pg_accessions",
+    "Gene names": "gg_names",
+    "PEP": "posterior_error_probability",
+    "Modified sequence": "peptidoform",
+    "Charge": "precursor_charge",
+    "Modified sequence": "peptidoform",
```

Ruff

Dictionary key literal `"Modified sequence"` repeated

Remove repeated key literal `"Modified sequence"`

(F601)